packages/service/src/infrastructure/configuration.ts

```diff
@@ -44,7 +50,9 @@ class Configuration {
       process.env.DISCORD_CLIENT_ID || '',
       process.env.DISCORD_GUILD_ID || '',
       process.env.DISCORD_TOKEN || '',
-      process.env.DROPBOX_TOKEN || '',
+      process.env.DROPBOX_CLIENT_ID || '',
+      process.env.DROPBOX_SECRET || '',
+      process.env.DROPBOX_REFRESH_TOKEN || '',
```

Comment:
I'm starting to feel like it's dumb to default to empty string when there is no value, but this is outside the scope of this PR.